crates/rust/src/lib.rs

```diff
@@ -377,9 +377,19 @@ impl InterfaceGenerator<'_> {
             "
                 #[allow(clippy::all)]
                 pub mod {snake} {{
+                    #[used]
+                    #[doc(hidden)]
+                    #[cfg(target_arch = \"wasm32\")]
+                    static __FORCE_SECTION_REF: fn() = __force_section_ref;
```

Comment:
Actually, I think we can simplify this to simply `crate::__force_section_ref`.  I'll fix.